torch/_dynamo/config.py

```diff
@@ -60,6 +60,7 @@
     torch.fx._symbolic_trace.is_fx_tracing: False,
     torch.onnx.is_in_onnx_export: False,
     external_utils.is_compiling: True,
+    torch._utils.is_compiling: True,
```

Comment:
How is that different from the one above: `external_utils.is_compiling` ?

Reply:
It actually isn't, but I spend like half the day Friday trying to resolve the circular dependency issue with importing dynamo in the optimizer code and couldn't resolve it. It came down to the config that dynamo uses importing a bunch of other torch modules which made it impossible to remove. Other workarounds I tried:

* inlining import (bad for performance)
* loading import once with a helper function (bad for performance)
* just calling torch._dynamo.is_compiling without importing like we do with is_scripting (caused weird import issues)

This ended up being the least intrusive, since this is temporary thought this was the best option.